.github/workflows/pr-build.yaml

```
@@ -0,0 +1,49 @@
+name: PR Build & Test
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/**'
+    branches:
+      - main
+
+env:
+  RUSTFLAGS: -Dwarnings
+
+jobs:
+  check:
+    runs-on: windows-2019
```

May be good to start with `windows-2022` if possible - that way you build with VS 2022 rather than VS 2019. 
```suggestion
    runs-on: windows-2022
```

---

Switching is a bit of a pain. 🥲 https://github.com/microsoft/windows-rs/pull/3056

---

You probably need VS 2022 for `NdrClientCall4` to fix the build issue.

---

I can help fast track https://github.com/microsoft/windows-rs/pull/3056 so that sudo can unblock on that, standby.

---

Heads up @zadjii-msft, we finished the switch on the windows-rs side of things.